spec/mocha.opts

```diff
@@ -0,0 +1,2 @@
+--require spec/setup.js
+--recursive
```

Comment:
![british Bender from futurama asking "whats all this then"](http://troll.me/images/compliment-bender/ello-whats-all-this-then.jpg)